dissect/target/tools/reg.py

```diff
@@ -38,20 +44,31 @@ def main():
 
     try:
         for target in Target.open_all(args.targets):
+            if not target.has_function("registry"):
+                log.error("Target %s has no Windows Registry", target)
```

Comment:
```suggestion
                target.log.error("Target has no Windows Registry")
```
I haven't tested how this looks, but it should include the target within the log message automatically.